### DISCUSSION TITLE
Game FPS caps

### DISCUSSION THREAD

**Discussion by Margen67** [xenia-canary/game-patches#71] [General]
Title ID | Name                                 | FPS                         | Patched
-------: | :---                                 | :--                         | :------
425307DB | WET                                  | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/425307DB%20-%20WET.patch)
43430841 | Resident Evil                        | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/43430841%20-%20Resident%20Evil.patch)
43430842 | Resident Evil 0                      | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/43430842%20-%20Resident%20Evil%200.patch)
434D0822 | Leisure Suit Larry Box Office Bust   | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/434D0822%20-%20Leisure%20Suit%20Larry.patch)
45410806 | Burnout Paradise (Screenshot)        | 60(?)                       | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/45410806%20-%20Burnout%20Paradise%20(Screenshot).patch)
45410809 | The Simpsons Game                    | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/45410809%20-%20The%20Simpsons%20Game.patch)
4541087F | Skate 2                              | 60(?)                       | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/4541087F%20-%20Skate%202.patch)
454108D8 | Army of Two: The 40th Day            | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/454108D8%20-%20Army%20of%20Two%20The%2040th%20Day.patch)
4541092A | Shadows of the Damned                | 31                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/4541092A%20-%20Shadows%20of%20the%20Damned.patch)
475007D2 | Hail to the Chimp                    | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/475007D2%20-%20Hail%20to%20the%20Chimp.patch)
4B4E0823 | Silent Hill Downpour                 | 30                          | ❎<sup>[1](https://github.com/xenia-canary/game-patches/blob/main/patches/4B4E0823%20-%20Silent%20Hill%20Downpour.patch) [2](https://github.com/xenia-canary/game-patches/blob/main/patches/4B4E0823%20-%20Silent%20Hill%20Downpour%20(TU1).patch)</sup>*<sup>See patch notes.</sup>
4D5307D1 | Project Gotham Racing 3              | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/4D5307D1%20-%20PGR3.patch)
4D5307D2 | Kameo Elements of Power              | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/4D5307D2%20-%20Kameo.patch)
4D5307D3 | Perfect Dark: Zero                   | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/4D5307D3%20-%20Perfect%20Dark%20Zero.patch)
4D5307DF | Blue Dragon                          | 30                          | [❎](https://github.com/xenia-canary/game-patches/blob/main/patches/4D5307DF%20-%20Blue%20Dragon.patch)*<sup>See patch notes.</sup>
4D5307E6 | Halo 3                               | 30                          | ❌<sup>Probably impossible due to physics timestep.</sup>
4D5307ED | Banjo-Kazooie Nuts                   | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/4D5307ED%20-%20Banjo-Kazooie%20Nuts.patch)
4D5307F1 | Fable II (GOTY/Platinum Edition)     | 30                          | ❎<sup>[1](https://github.com/xenia-canary/game-patches/blob/main/patches/4D5307F1%20-%20Fable%20II%20(GOTY_Platinum%20Edition).patch) [2](https://github.com/xenia-canary/game-patches/blob/main/patches/4D5307F1%20-%20Fable%20II%20(GOTY_Platinum%20Edition%2C%20TU1).patch)</sup>*<sup>See patch notes.</sup>
4D5307F2 | Viva Piñata                          | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/4D5307F2%20-%20Viva%20Pinata.patch)
4D5307FA | Lost Odyssey                         | 30                          | ❎<sup>[1](https://github.com/xenia-canary/game-patches/blob/main/patches/4D5307FA%20-%20Lost%20Odyssey.patch) [2](https://github.com/xenia-canary/game-patches/blob/main/patches/4D5307FA%20-%20Lost%20Odyssey%20(NTSC-J).patch)</sup>*<sup>See patch notes.</sup>
4D53085B | Halo: Reach                          | 30                          | ❌
4D53085F | Viva Piñata: Trouble In Paradise     | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/4D53085F%20-%20Viva%20Pinata%20TIP.patch)
4D530877 | Halo 3: ODST                         | 30                          | ❌
4D5387E0 | Kameo Elements of Power (kiosk demo) | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/4D5387E0%20-%20Kameo%20(kiosk%20demo).patch)
4E4D07E4 | Eternal Sonata                       | 30 <sup>(60 in menus)</sup> | [❎](https://github.com/xenia-canary/game-patches/blob/main/patches/4E4D07E4%20-%20Eternal%20Sonata.patch)*<sup>See patch notes.</sup>
4D5307F9 | Project Gotham Racing 4              | 30                          | [❎](https://github.com/xenia-canary/game-patches/blob/main/patches/4D5307F9%20-%20PGR4.patch)*<sup>See patch notes.</sup>
4E4D07F0 | Splatterhouse                        | 30                          | ❌*<sup>Framerate limited by Havok.</sup>
535107E4 | Final Fantasy 13                     | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/535107E4%20-%20Final%20Fantasy%2013.patch)
535107E8 | NIER                                 | 30                          | [❎](https://github.com/xenia-canary/game-patches/blob/main/patches/535107E8%20-%20NIER.patch)*<sup>See patch notes.</sup>
53510806 | Final Fantasy 13-2                   | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/53510806%20-%20Final%20Fantasy%2013-2.patch)
53518810 | Final Fantasy 13-2 Demo              | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/53518810%20-%20Final%20Fantasy%2013-2%20Demo.patch)
545107D1 | Saints Row                           | 30                          | ❌<sup>Probably impossible due to physics timestep.</sup>
545107D2 | The Outfit                           | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/545107D2%20-%20The%20Outfit.patch)
545107F1 | Ratatouille                          | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/545107F1%20-%20Ratatouille.patch)
545407E0 | Prey                                 | Camera updates at 30 FPS.   | ❌
545407F8 |                                      | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/545407F8%20-%20Midnight%20Club%20Los%20Angeles.patch)*<sup>See patch notes.</sup>
5454082B | GOTY                                 | 30                          | ✅<sup>[1](https://github.com/xenia-canary/game-patches/blob/main/patches/5454082B%20-%20Red%20Dead%20Redemption%20(GOTY%2C%20Disc%201).patch) [2](https://github.com/xenia-canary/game-patches/blob/main/patches/5454082B%20-%20Red%20Dead%20Redemption%20(GOTY%2C%20Disc%202).patch)</sup>*<sup>See patch notes.</sup>
|        | Original                             | 30                          | ✅<sup>[1](https://github.com/xenia-canary/game-patches/blob/main/patches/5454082B%20-%20Red%20Dead%20Redemption%20(Original%2C%20NTSC).patch) [2](https://github.com/xenia-canary/game-patches/blob/main/patches/5454082B%20-%20Red%20Dead%20Redemption%20(Original%2C%20NTSC%2C%20TU9).patch)</sup>*<sup>See patch notes.</sup>
5841090B | Doritos Dash of Destruction          | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/5841090B%20-%20Dash%20of%20Destruction.patch)
58410954 | Banjo-Kazooie                        | 30                          | [❎](https://github.com/xenia-canary/game-patches/blob/main/patches/58410954%20-%20Banjo%20Kazooie.patch)*<sup>See patch notes.</sup>
58410955 | Banjo-Tooie                          | 30                          | [❎](https://github.com/xenia-canary/game-patches/blob/main/patches/58410955%20-%20Banjo%20Tooie.patch)*<sup>See patch notes.</sup>
58410A70 | Harm's Way                           | 30                          | [✅](https://github.com/xenia-canary/game-patches/blob/main/patches/58410A70%20-%20Harm's%20Way.patch)

**Comment by bomabomaboma** [comment]
545407E0 - Prey - Camera updates at 30 FPS - Not patched
545107D1 - Saints Row - 30 FPS - Not patched (probably impossible to do so)
4D5307E6 - Halo 3 - 30 FPS - Not patched (again, probably impossible due to physics timestep)

**Comment by bomabomaboma** [comment]
4E4D07F0 - Splatterhouse - 30 FPS - Not patched (framerate limited by Havok)
4541092A - Shadows of the Damned - 31 FPS - Patched

**Comment by Etokapa** [comment]
More games with FPS caps with vsync off:

| Title ID | Title                  | FPS Limit | Notes |
|----------|------------------------|-----------|-------|
| 5A4907D1 | Rambo                  | 30        |       |
| 45410916 | Alice: Madness Returns | 58        | UE3 - [Patched ](https://github.com/xenia-canary/game-patches/blob/main/patches/45410916%20-%20Alice%20Madness%20Returns.toml) |
| 5345080E | Alpha Protocol         | 62        | UE3   |
| 434307D4 | Resident Evil 5        | 30        | MTF2   |
| 43430814 | Dragon's Dogma       | 30        | MTF2   |
| 43430814 | Dragon's Dogma: Dark Arisen       | 30        | MTF2   |
| 584113DB | Marvel Puzzle Quest: Dark Reign       | 30        |    |


Will update this comment if I find more.

**Comment by Margen67** [comment]
This info should be moved here: https://github.com/xenia-canary/xenia-canary.github.io
